Explorer++/Helper/FileOperations.cpp

```diff
@@ -46,12 +46,24 @@ HRESULT FileOperations::RenameFile(IShellItem *item, const std::wstring &newName
 	return hr;
 }
 
-HRESULT FileOperations::DeleteFiles(HWND hwnd, const std::vector<PCIDLIST_ABSOLUTE> &pidls,
-	bool permanent, bool silent)
+HRESULT FileOperations::DeleteFiles(HWND hwnd, std::vector<PCIDLIST_ABSOLUTE> &pidls, bool permanent, bool silent)

```

Comment:
It looks like there are formatting changes here. Could you make sure that you format the code using ClangFormat.  Visual Studio has [built-in support](https://learn.microsoft.com/en-us/visualstudio/ide/reference/options-text-editor-c-cpp-formatting?view=vs-2022) for ClangFormat, so you should be able to format from within the IDE.